backend/api/Database/Context/InitDb.cs

```diff
@@ -87,7 +87,7 @@ private static List<Mission> GetMissions()
             EchoMissionId = 95,
             IsarMissionId = "1",
             MissionStatus = MissionStatus.Pending,
-            DesiredStartTime = DateTimeOffset.UtcNow.AddHours(3),
+            DesiredStartTime = DateTimeOffset.UtcNow,
```

Comment:
Like @aestene mentioned, maybe we want to keep these ass scheduled for a couple of hours after startup?
We used to have them start right away and it was sometimes hard to test new things if they started and crashed right away

Reply:
Status of these missions has been changed to successful to prevent them from starting